primitives/src/crh/poseidon/mod.rs

```diff
@@ -706,22 +646,19 @@ mod test {
 
     #[test]
     fn test_hash_speed() {
```

Comment:
Please use the bench framework to write this kind of test, is very useful and more accurate